examples/pylab_examples/cursor_demo.py

```diff
@@ -1,17 +1,19 @@
-#!/usr/bin/env python
 # -*- noplot -*-
 
 """
-
 This example shows how to use matplotlib to provide a data cursor.  It
 uses matplotlib to draw the cursor and may be a slow since this
 requires redrawing the figure with every mouse move.
 
 Faster cursoring is possible using native GUI drawing, as in
-wxcursor_demo.py
+wxcursor_demo.py. 
```

Comment:
Unfortunately there is a trailing whitespace here now causing the pep8 test to fail